src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -193,15 +194,15 @@ def main():
     )
 
     if verbose:
-        print("supernetwork connections set complete")
+        logger.warning("supernetwork connections set complete")
     if showtiming:
-        print("... in %s seconds." % (time.time() - start_time))
+        logger.warning("... in %s seconds." % (time.time() - start_time))
 
     # STEP 2
     if showtiming:
         start_time = time.time()
     if verbose:
-        print("organizing connections into reaches ...")
+        logger.warning("organizing connections into reaches ...")
```

Comment:
I'll make those changes. I just left the print outs from --verbose as warning print outs to stay consistent.

Reply:
I see. I would also suggest removing the checks for verbose. The logging library controls verbosity by setting the log level. The verbose flag should be determining the log level and nothing else.

Reply:
Okay, would it be beneficial to remove the debug-level as well and only specify the log level from the command line arguments?  It seems pointless to have both of them when this should replace it.

Reply:
You can still have the user specify the debug-level and verbosity and use that to determine what log level should be. After that is done, there is no need to do any further checking.
When you log a message, the very first thing that the library checks is the logging level. It will only continue to process messages that are greater than or equal to the log level. This removes the need for checking in your application code as you're offloading the work to the logging library.

Reply:
See: https://docs.python.org/3/howto/logging.html#when-to-use-logging